Upload step for copy process completion added (#146)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/usermanagement-manage.adoc

== Reset a User Password

> Reset a password if the user forgot it. RPA Manager sends an email containing a new temporary password to the user. The user can then create a new password.

Reviewer: ```suggestion
Reset a password if the user forgot it. RPA Manager sends an email that contains a new temporary password to the user. The user can then create a new password.
```

. Open the *Users* view of the *User Management* module.
. Click the *Reset Password* icon image:resetpassword-icon.png[key symbol,1.5%,1.5%] in the panel of the user to edit.

== Change Your Password

> Change your own password regularly. You are logged out in the process and have to log in again. The password is used for login to RPA Manager and RPA Builder.

Reviewer: ```suggestion
Change your own password regularly. You are logged out of the process and have to log in again. The password is used to log in to RPA Manager and RPA Builder.
```

Author: You are logged out in the process [of changing your password] and have to log in again.
This texts will become obsolete soon.

Designate users responsible for the automation of processes as project managers to enable them to approve of a process candidate or to start an automation.

. Open the *Users* view of the *User Management* module.

> . Click the *Edit* icon image:edit-icon.png[pen-to-paper symbol,1.5%,1.5%] in the panel of the user you want to designate as project manager.

Reviewer: ```suggestion
. Click the *Edit* icon image:edit-icon.png[pen-to-paper symbol,1.5%,1.5%] in the panel of the user that you want to designate as project manager.
```
